application/perfect/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM swift:latest
+
+RUN apt-get update
+
+RUN apt-get upgrade -y && \
+    apt-get install -y git
+
+RUN apt-get install -y openssl libssl-dev uuid-dev
```

Comment:
What exactly are these dependencies needed for?

Reply:
According to the official website, when it comes to Ubuntu, `Perfect relies on OpenSSL, libssl-dev, and uuid-dev.` You can find more details below:
```
https://perfect.org/docs/gettingStarted.html
```